broadinstitute/orsp-pub
# Remove actor from query to find the person to request access to a Project or CG

## Addresses
- https://broadinstitute.atlassian.net/browse/BTRX-738

## Changes
- Remove actor from query to find user to request access to a Project/CG
- Fix validation in Awaiting Action From <p> 

## Testing
- Search for Projects and Consent Groups from the top nav or from the Search page and the actor should never appear as the person to request access.
- Enter a CG and Awaiting action From should not appear

---
